### PR TITLE
NGDOC-174 Add anycast feature to FAQ

### PR DESCRIPTION
**Pull request by Jianghai-Quantil**
@nibinqtl @burtonquantil @marcfishelquantil Please help review, thanks.

### CODE REVIEW

File: en/docs/edge-logic/faq.md [hunk 1]
```diff
@@ -133,6 +133,14 @@ In many cases, "dynamic" does not mean the content is not cacheable. For example
...
+
+However, if your domain to be accelerated is an Apex domain, i.e. a root domain (e.g. example.com) that does not contain a subdomain, adding a CNAME record to the Apex domain would become an issue, due to restrictions in DNS specification (Refer to [RFC1034](https://www.ietf.org/rfc/rfc1034.txt)). This is where our anycast traffic routing feature comes into play. It provides anycast IPv4 addresses which you can enter into your DNS as A records to point your Apex domain to CDN Pro. Each of the anycast IP addresses is broadcast in multiple locations, basically the great majority of our global network of PoPs except those in mainland China. A request targeting an anycast IP address is directed to a PoP that is nearest to where the request is originated. This ensures that your content is served to end users by locations as close as possible.
```
Comment (nibinqtl) [inline en/docs/edge-logic/faq.md:140]: The purpose of our Anycast IP is to provide a highly available address for the Apex domain, and the edge logic should simply respond with a 30x redirection to the "www" domain. The Anycast IP includes only a small portion of our PoPs so the performance and capacity is not comparable to the edge hostnames. It should be used only for Apex domains.

**Review by nibinqtl — CHANGES_REQUESTED** [review]

File: en/docs/edge-logic/faq.md (hunk 1 above)
Comment (Jianghai-Quantil) [reply en/docs/edge-logic/faq.md]: @nibinqtl Thanks for the comment. Just make some changes. Please review again.

File: en/docs/edge-logic/faq.md [hunk 2]
```diff
@@ -133,6 +133,14 @@ In many cases, "dynamic" does not mean the content is not cacheable. For example
...
 
+### Apex domains and anycast
+
+Our [CNAME-based traffic management system](</docs/portal/traffic-management/overview>) is designed to make routing decisions dynamically, factoring in performance, costs, compliance requirements and custom rules, etc. Simply [create an edgeHostname](</docs/portal/traffic-management/creating-edge-hostname>) and add a CNAME record to your DNS to point your domain at the edgeHostname, then let CDN Pro do the magic. In most cases, you can leverage the powerful capabilities of this system to deliver optimal results. 
+
+However, if your domain to be accelerated is an Apex domain, i.e. a root domain (e.g. example.com) that does not contain a subdomain, adding a CNAME record to the Apex domain would become an issue, due to restrictions in DNS specification (Refer to [RFC1034](https://www.ietf.org/rfc/rfc1034.txt)). This is where our anycast addressing feature comes into play. It provides anycast IPv4 addresses which you can enter into your DNS as A records to point your Apex domain to some PoPs of CDN Pro. This feature is currently available on demand. Please contact our support team if you need to use it.
```
Comment (burtonquantil) [inline en/docs/edge-logic/faq.md:140]: Are you referring to a particular section of the RFC 1034 ?

File: en/docs/edge-logic/faq.md [hunk 3]
```diff
@@ -133,6 +133,14 @@ In many cases, "dynamic" does not mean the content is not cacheable. For example
...
+
+Note that each anycast IP address is broadcast only in a small subset of our global PoPs. And the list of PoPs where anycast is available might be subject to changes without prior notice. So the performance and capacity behind an anycast IP address are not comparable to that of an edgeHostname. We recommend using the CNAME-based edgeHostnames to route traffic, as much as possible. Anycast should be used only for Apex domains.
```
Comment (burtonquantil) [inline en/docs/edge-logic/faq.md:142]: I'd combine the first two sentences.

Note that each anycast IP address is broadcast only in a small subset of our global PoPs, and the list of PoPs where anycast is available is subject to changes without prior notice. Therefore, the performance  and capacity behind an anycast IP address are not comparable to that of an edge  hostname.   We recommend using the CNAME-based edge hostnames to route traffic as much as possible. Anycast should be used only for Apex domains.

File: en/docs/edge-logic/faq.md (hunk 2 above)
Comment (burtonquantil) [reply en/docs/edge-logic/faq.md:140]: Based on what I've seen on other companies' sites, I don't believe it is necessary to capitalize "apex" as "Apex".

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I posted questions and suggested a change to improve the grammar

**Comment by Jianghai-Quantil** [comment]
Descriptions revised per @burtonquantil 's comments.

File: en/docs/edge-logic/faq.md [hunk 4]
```diff
@@ -133,6 +133,14 @@ In many cases, "dynamic" does not mean the content is not cacheable. For example
...
 
+### Apex domains and anycast
+
+Our [CNAME-based traffic management system](</docs/portal/traffic-management/overview>) is designed to make routing decisions dynamically, factoring in performance, costs, compliance requirements and custom rules, etc. Simply [create an edgeHostname](</docs/portal/traffic-management/creating-edge-hostname>) and add a CNAME record to your DNS to point your domain at the edgeHostname, then let CDN Pro do the magic. In most cases, you can leverage the powerful capabilities of this system to deliver optimal results.
```
Comment (burtonquantil) [inline en/docs/edge-logic/faq.md:138]: I recommend removing ", etc." because it is vague.

Comment (burtonquantil) [inline en/docs/edge-logic/faq.md:138]: "edgeHostname" should be "edge hostname". That is what we use in the rest of the documentation.

File: en/docs/edge-logic/faq.md [hunk 5]
```diff
@@ -133,6 +133,14 @@ In many cases, "dynamic" does not mean the content is not cacheable. For example
...
 
+### Apex domains and anycast
+
+Our [CNAME-based traffic management system](</docs/portal/traffic-management/overview>) is designed to make routing decisions dynamically, factoring in performance, costs, compliance requirements and custom rules, etc. Simply [create an edgeHostname](</docs/portal/traffic-management/creating-edge-hostname>) and add a CNAME record to your DNS to point your domain at the edgeHostname, then let CDN Pro do the magic. In most cases, you can leverage the powerful capabilities of this system to deliver optimal results. 
+
+However, if your domain to be accelerated is an apex domain, i.e. a root domain (e.g. example.com) that does not contain a subdomain, adding a CNAME record to the apex domain would become an issue, due to restrictions in the DNS specification (Refer to section 3.6.2 of [RFC1034](https://www.ietf.org/rfc/rfc1034.txt)). This is where our anycast addressing feature comes into play. It provides anycast IPv4 addresses which you can enter into your DNS as A records to point your apex domain to some PoPs of CDN Pro. This feature is currently available on demand. Please contact our support team if you need to use it.
```
Comment (burtonquantil) [inline en/docs/edge-logic/faq.md:140]: I suggest changing "CNAME record to the apex domain" to "CNAME record for the apex domain".

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Thanks for the revisions. I made a few more suggestions.

File: en/docs/edge-logic/faq.md (hunk 4 above)
Comment (Jianghai-Quantil) [reply en/docs/edge-logic/faq.md]: Removed

Comment (Jianghai-Quantil) [reply en/docs/edge-logic/faq.md]: Changed to edge hostname.

File: en/docs/edge-logic/faq.md (hunk 5 above)
Comment (Jianghai-Quantil) [reply en/docs/edge-logic/faq.md]: Changed as suggested

File: en/docs/edge-logic/faq.md [hunk 6]
```diff
@@ -133,6 +133,14 @@ In many cases, "dynamic" does not mean the content is not cacheable. For example
...
 
+### Apex domains and anycast
+
+Our [CNAME-based traffic management system](</docs/portal/traffic-management/overview>) is designed to make routing decisions dynamically, factoring in performance, costs, compliance requirements and custom rules. Simply [create an edge hostname](</docs/portal/traffic-management/creating-edge-hostname>) and add a CNAME record to your DNS to point your domain at the edge hostname, then let CDN Pro do the magic. In most cases, you can leverage the powerful capabilities of this system to deliver optimal results.
```
Comment (burtonquantil) [inline en/docs/edge-logic/faq.md:138]: Minor: I prefer a comma after "requirements" before "and",  though it's technically still OK as is.

I prefer to replace "In most cases, you can leverage" with a stronger sounding  "Leverage"

**Review by burtonquantil — DISMISSED** [review]
I made a couple of minor suggestions, but it is OK to merge.

File: en/docs/edge-logic/faq.md (hunk 6 above)
Comment (Jianghai-Quantil) [reply en/docs/edge-logic/faq.md]: Suggestions accepted.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**